spec/controllers/job_definitions_controller_spec.rb

```diff
@@ -127,14 +129,30 @@
 
   describe '#update' do
     let(:admin) { create(:user) }
+    let(:script) { "noop:\n" }
     subject! do
-      patch :update, params: { id: definition.id, job_definition: { name: 'Job Definition', description: 'This is description', script: "noop:\n" }, admin_assignments: { user_id: ["", admin.id] } }
+      patch :update, params: { id: definition.id, job_definition: { name: 'Job Definition', description: 'This is description', script: script }, admin_assignments: { user_id: ["", admin.id] } }
     end
 
     it do
       expect(response).to redirect_to(assigns(:definition))
 
       expect(assigns(:definition)).not_to be_new_record
+      expect(assigns(:definition).revisions.size).to eq 1
+    end
+
+    describe 'changes script' do
+      let(:script) { "noop:\nnoop:\n" }
+      it do
+      expect(response).to redirect_to(assigns(:definition))
+
+      expect(assigns(:definition)).not_to be_new_record
+      expect(assigns(:definition).script).to eq script
+      expect(assigns(:definition).revisions.size).to eq 2
+      expect(assigns(:definition).revisions.first.script).to eq script
+      expect(assigns(:definition).revisions.first.user).to eq controller.current_user
+
```

Comment:
empty line :eyes:

Reply:
Thank you. I fixed in https://github.com/cookpad/kuroko2/pull/110/commits/12e87faf0ccb6e76e6b30d07c4324ca0e33cb60b.